glue/viewers/matplotlib/state.py

```diff
@@ -200,16 +200,20 @@ class MatplotlibLayerState(LayerState):
                                                    'the data')
     alpha = DeferredDrawCallbackProperty(docstring='The transparency used to '
                                                    'display the data')
+    preferred_cmap = DeferredDrawCallbackProperty(docstring='The preferred color used to display '
+                                                            'the data')
```

Comment:
As before, don't add this as a callback property, just access it on ``layer.style`` whenever it is needed.

Reply:
Okay, sure. Deleted.